iree/tools/benchmark_module_main.cc

```diff
@@ -147,6 +147,7 @@ Status Run(::benchmark::State& state) {
                                  iree_allocator_system()));
   }
 
+  inputs.reset();
```

Comment:
Should we similarly be releasing `outputs` in the scopes above?

Reply:
I think `outputs` is released when existing the for-loop because it's a local variable.

Reply:
Ah, but this has to happen before the `iree_*_release` calls?

Reply:
Yes. Otherwise, the allocator will be destroyed before the buffer.